src/functionalTest/kotlin/kotlinx/kover/test/functional/cases/utils/Defaults.kt

```diff
@@ -1,14 +1,24 @@
 package kotlinx.kover.test.functional.cases.utils
 
+import kotlinx.kover.api.*
+import kotlinx.kover.test.functional.core.ProjectType
 
-internal const val DEFAULT_INTELLIJ_KJVM_BINARY = "kover/test.ic"
-internal const val DEFAULT_INTELLIJ_KJVM_SMAP = "kover/test.ic.smap"
 
-internal const val DEFAULT_INTELLIJ_KMP_BINARY = "kover/jvmTest.ic"
-internal const val DEFAULT_INTELLIJ_KMP_SMAP = "kover/jvmTest.ic.smap"
+internal fun defaultBinaryReport(engine: CoverageEngine, projectType: ProjectType): String {
+    val extension = if (engine == CoverageEngine.INTELLIJ) "ic" else "exec"
+    return when (projectType) {
+        ProjectType.KOTLIN_JVM -> "kover/test.$extension"
+        ProjectType.KOTLIN_MULTIPLATFORM -> "kover/jvmTest.$extension"
+        ProjectType.ANDROID -> "kover/jvmTest.$extension"
+    }
+}
 
-internal const val DEFAULT_JACOCO_KJVM_BINARY = "kover/test.exec"
-internal const val DEFAULT_JACOCO_KMP_BINARY = "kover/jvmTest.exec"
+internal fun defaultSmapFile(projectType: ProjectType): String {
+    return defaultBinaryReport(CoverageEngine.INTELLIJ, projectType) + ".smap"
+}
 
-internal const val DEFAULT_XML = "reports/kover/report.xml"
-internal const val DEFAULT_HTML = "reports/kover/html"
+internal fun defaultXmlReport() = "reports/kover/report.xml"
```

Comment:
This is done because at the moment there is no certainty that additional parameters will not be needed to get the default value (for example, as in the case of defaultBinaryReport). So it turns out the general style that getting all the values is a functions, and not a mixture of `val` and `fun`.

However, I do not insist on such a decision, if `val` is perceived better, I can change it.